pkgs/development/python-modules/praw/default.nix

```diff
@@ -15,7 +15,7 @@
 
 buildPythonPackage rec {
   pname = "praw";
-  version = "7.5.0";
+  version = "7.6.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
```

Comment:
The release notes say:

```
Drop support for Python 3.6, which is end-of-life on 2021-12-23.
```

Does this mean this should now be `pythonOlder "3.7"`?

Reply:
Yes, thanks for checking.